contracts/staking/GoodFundManager.sol

```diff
@@ -60,7 +63,15 @@ contract GoodFundManager is DAOContract {
     // Last block number which `transferInterest`
     // has been executed in
     uint256 public lastTransferred;
-
+    //Structure that hold reward information and if its blacklicksted or not for particular staking Contract
+    struct Reward{
+        uint32 blockReward; //in G$
+        uint32 blockStart;
+        uint32 blockEnd;
+        bool isBlackListed;
+    }
+    // Rewards per block for particular Staking contract
+    mapping(address => Reward) rewardsForStakingContract;
```

Comment:
in general make variables public unless they dont hold any interesting or trivial information